test.js

```diff
@@ -62,3 +62,9 @@ it('should support `--color=false` flag', function () {
 	process.argv = ['--color=false'];
 	assert.equal(requireUncached('./'), false);
 });
+
+it('should ignore post-terminator flags', function () {
+	process.env.COLORTERM = true; // force it to say yes unless we say otherwise
```

Comment:
Because of this you're hitting the if/return statement in `index.js` at line 11, and never asserting the `--` behaviour.

I suggest removing this line, and adding `'--color'` as the first argument in the array literal in the line below. The assertion will remain the same, but now you're actually hitting the branch you want to test.